ui/src/routes/results.tsx

```diff
@@ -50,6 +50,8 @@ function Results() {
     setPageSize(size);
   };
 
+  window.scrollTo(0, 0);
+
```

Comment:
why do we need this?

Reply:
Because it should bring the user to the top of the page.